biz.aQute.bndlib/src/aQute/bnd/classfile/Attribute.java

```diff
@@ -12,6 +12,54 @@
 
 @ProviderType
 public interface Attribute {
+
+	@ProviderType
+	enum AttributeTag {
```

Comment:
The problem they solve is that when you introduce a new type, the places where you depend on types will fire a warning. i.e. if you add a new Attribute, the methods where I depend on analyzing the attributes need to be inspected.

I understand that you want to have a preview phase, but that as a good example where this will help finding the places where there are dependencies.